packages/amplication-server/src/core/action/action.service.ts

```diff
@@ -111,6 +110,31 @@ export class ActionService {
     });
   }
 
+  completeWithLog(step: ActionStep, userActionId: string, topicName: string) {
```

Comment:
@abrl91 I think this is very confusing for future readers of this code that there are so many ways to send a log and/or complete the actionStep, either via completeWithLog or via emitting the message with isComplete=true.
Since we decided that we want the step completion to only be done on the consumer side, let's remove this function on the producer side in order to clarify the code.